lib/devicetrust/authn/authn.go

```diff
@@ -220,15 +233,22 @@ func (c *Ceremony) authenticateDeviceMacOS(
 func (c *Ceremony) authenticateDeviceTPM(
 	stream devicepb.DeviceTrustService_AuthenticateDeviceClient,
 	resp *devicepb.AuthenticateDeviceResponse,
+	sshSigner *keys.PrivateKey,
 ) error {
-	challenge := resp.GetTpmChallenge()
```

Comment:
Why the variable name change?
I think it was better before because now we have a `chalResp` and a `challengeResponse` which have the same meaning?

Reply:
The reasoning was that `challenge` now conflicts with the package of the same name, and this is called `chalResp` in `authenticateDeviceMacOS` above. But that is quite confusing with `challengeResponse` here also, good point. Renamed this to `tpmChallenge`.